go.mod

```diff
@@ -77,6 +77,7 @@ require (
 	github.com/terraform-providers/terraform-provider-azurestack v0.10.0
 	github.com/terraform-providers/terraform-provider-ignition/v2 v2.1.0
 	github.com/terraform-providers/terraform-provider-local v1.4.0
+	github.com/terraform-providers/terraform-provider-nutanix v1.1.0
```

Comment:
Why is this needed for this PR?

Reply:
This is used to create a client that is used in later PRs.